app/src/main/java/com/imdbmovieapp/domain/usecase/deleteUseCase/DeleteUseCase.kt

```diff
@@ -0,0 +1,13 @@
+package com.imdbmovieapp.domain.usecase.deleteUseCase
+
+import com.imdbmovieapp.domain.base.BaseUseCase
+import com.imdbmovieapp.domain.model.MovieDomain
+import com.imdbmovieapp.domain.repository.MovieRepository
+
+class DeleteUseCase(
```

Comment:
when you will change functions name, please change useCase names also. `DeleteUseCase` i cant understand class functionality from name